src/sdk/namespace_fs.js

```diff
@@ -309,9 +344,11 @@ class NamespaceFS {
                         return;
                     }
 
+                    const common_prefix = await is_directory_or_symlink_to_directory(ent, fs_account_config, path.join(dir_path, ent.name));
```

Comment:
I would change it to isDir like you do in other places - sounds less confusing

Reply:
I meant: 
```suggestion
                    const isDir = await is_directory_or_symlink_to_directory(ent, fs_account_config, path.join(dir_path, ent.name));
```

Reply:
Done